src/brag/client/ceph-brag

```diff
@@ -258,7 +258,7 @@ def get_nums():
   num_mons = len(oj['monmap']['mons'])
   num_osds = int(oj['osdmap']['osdmap']['num_in_osds'])
   try:
-    num_mdss = oj['mdsmap']['in']
+    num_mdss = oj['fsmap']['in']
```

Comment:
@zphj1987 could you fill more details in the commit message? the one in https://github.com/ceph/ceph/pull/13079/commits looks great, IMO.